auditbeat/scripts/docs_collector.py

```diff
@@ -42,21 +42,39 @@ def collect(base_paths):
         with open(beat_path + "/fields.yml") as f:
             fields = yaml.load(f.read(), Loader=yaml.FullLoader)
             title = fields[0]["title"]
+            applies_to = ""
+            if "version" in fields[0]:
+                version = fields[0]["version"]
+                versions = []
+                for key, value in version.items():
+                    versions.append(f"{key} {value}")
+                applies_to = ", ".join(versions)
```

Comment:
I was skeptical that this would make the applies_to string properly when there's 0 or 1 items in versions, because I wasn't sure about the python behaviour, but I confirmed it works properly. With 0 items, its an empty list, and with one item, there is just the item, with no extra comma or spaces.

Reply:
I'm also not very experienced with Python so there definitely could be a better way to do this, but I also tested several different scenarios and got the expected output.